docs/design/fabric-v2+/fpc-lifecycle-v2.puml

```diff
@@ -85,7 +85,7 @@ ref over Admin, Peer_CLI, Peer2
 end
 
 group approveformyorg
-    Admin -> Peer_CLI++ : lifecycle\nchaincode approveformyorg\n...**--version mrenclave**...
+    Admin -> Peer_CLI++ : lifecycle\nchaincode approveformyorg\n(version=**mrenclave**)
```

Comment:
Not sure why you changed my original proposal (which matched the parameters of the cli) to the current version which seems to me a bit less clear but i have no strong feeling